CODEOWNERS

```diff
@@ -113,8 +113,9 @@
 /src/lib/snark_bits/ @bkase
 /src/lib/snark_work_lib/ @deepthiskumar
 /src/lib/snark_worker/ @cmr @nholland94
+/src/lib/snark_keys/ @imeckler @mrmr1993
 /src/lib/snarkette/ @bkase
-# /src/lib/snarky (submodule!)
+/src/lib/snarky @imeckler @mrmr1993
```

Comment:
this should have a trailing slash I think?

Reply:
Done, thanks!